src/OnlySpans.PolyLeads.Api/Data/Entities/DocumentConfiguration.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace OnlySpans.PolyLeads.Api.Data.Entities;
+
+public class DocumentConfiguration : IEntityTypeConfiguration<Document>
```

Comment:
```suggestion
public sealed class DocumentConfiguration : IEntityTypeConfiguration<Document>
```

это везде так, кроме Entity